packages/examples/packages/invoke-snap/packages/consumer-signer/src/index.ts

```diff
@@ -52,7 +52,7 @@ export const onRpcRequest: OnRpcRequestHandler = async ({ request }) => {
           request: {
             method: 'getAccount',
             params: {
-              path,
+              path: path as unknown as string[],
```

Comment:
Why do we need to recast this?

Reply:
Because `PartialHDPathTuple5` used for `path` is an `readonly` array of constant strings.

`readonly` types can't be assigned to `JSON` type.

Readonly types can't be recast into mutable types without recasting into `unknown` beforehand.